GenespotRE/settings.py

```diff
@@ -294,17 +282,34 @@ def GET_BQ_COHORT_SETTINGS():
     'allauth.socialaccount',
     'allauth.socialaccount.providers.google')
 
-TEMPLATE_CONTEXT_PROCESSORS = (
-    'allauth.socialaccount.context_processors.socialaccount',
-    'django.core.context_processors.request',
-    'django.contrib.auth.context_processors.auth',
-    'django.contrib.messages.context_processors.messages',
-    'django.core.context_processors.tz'
-)
-
-TEMPLATE_DIRS += (
-    os.path.join(BASE_DIR, 'templates', 'accounts'),
-    )
+# Template Engine Settings
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'OPTIONS': {
+            # add any context processors here
+            'context_processors': (
+                'allauth.socialaccount.context_processors.socialaccount',
```

Comment:
I just noticed something--although we install allauth.account as an app, we don't use its context processor ('allauth.account.context_processors.account'). Is this intentional?